batch_conversion.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Seems not to be compatible with python2 (this is not a problem, we don't really care of python2 anymore).